addons/account_sequence/account_sequence_installer.py

```diff
@@ -1,80 +1,48 @@
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
 
-from openerp.osv import fields, osv
+from openerp import api, fields, models
 
-class account_sequence_installer(osv.osv_memory):
+
+class AccountSequenceInstaller(models.Model):
     _name = 'account.sequence.installer'
     _inherit = 'res.config.installer'
 
-    _columns = {
-        'name': fields.char('Name', required=True),
-        'prefix': fields.char('Prefix', size=64, help="Prefix value of the record for the sequence"),
-        'suffix': fields.char('Suffix', size=64, help="Suffix value of the record for the sequence"),
-        'number_next': fields.integer('Next Number', required=True, help="Next number of this sequence"),
-        'number_increment': fields.integer('Increment Number', required=True, help="The next number of the sequence will be incremented by this number"),
-        'padding' : fields.integer('Number padding', required=True, help="Odoo will automatically adds some '0' on the left of the 'Next Number' to get the required padding size."),
-        'company_id': fields.many2one('res.company', 'Company'),
-    }
-    _defaults = {
-        'company_id': lambda s,cr,uid,c: s.pool.get('res.company')._company_default_get(cr, uid, 'ir.sequence', context=c),
-        'number_increment': 1,
-        'number_next': 1,
-        'padding' : 0,
-        'name': 'Internal Sequence Journal',
-    }
 
-    def execute(self, cr, uid, ids, context=None):
-        if context is None:
-            context = {}
-        record = self.browse(cr, uid, ids, context=context)[0]
-        j_ids = []
-        if record.company_id:
-            company_id = record.company_id.id,
-            search_criteria = [('company_id', '=', company_id)]
-        else:
-            company_id = False
-            search_criteria = []
-        vals = {
-            'id': 'internal_sequence_journal',
-            'code': 'account.journal',
-            'name': record.name,
-            'prefix': record.prefix,
-            'suffix': record.suffix,
-            'number_next': record.number_next,
-            'number_increment': record.number_increment,
-            'padding' : record.padding,
-            'company_id': company_id,
-        }
+    name = fields.Char(string='Name', required=True, default='Internal Sequence Journal')
+    prefix = fields.Char(string='Prefix', help="Prefix value of the record for the sequence")
+    suffix = fields.Char(string='Suffix', help="Suffix value of the record for the sequence")
```

Comment:
You can remove the string parameter here, as the ORM will generate a capitalized version of the field name by default.